chore: enable stalebot

benoitf opened the pull request:
### What does this PR do?
still using a low operations and high range for days
will decrease it after some runs

based on the log of https://github.com/containers/podman-desktop/actions/runs/7928011471 it should only flag few issues

### Screenshot / video of UI

<!-- If this PR is changing UI, please include 
screenshots or screencasts showing the difference -->

### What issues does this PR fix or reference?

<!-- Include any related issues from Podman Desktop 
repository (or from another issue tracker). -->

### How to test this PR?

<!-- Please explain steps to reproduce -->

jeffmaury commented:
Can we keep 30 days before closing the issue. 2 weeks seems a little low to me if you're PTO

benoitf commented:
@jeffmaury updated

jeffmaury reviewed (approved):
LGTM

benoitf commented:
ok let see what is happening, merging

benoitf commented:
example of action: https://github.com/containers/podman-desktop/issues/476#issuecomment-1948366278

benoitf commented:
```

Statistics:
Processed items: 777
├── Processed issues: 717
└── Processed PRs   : 60
New stale issues: 1
Added issues labels: 1
Added issues comments: 1
Fetched items: 786
Fetched items events: 1
Fetched items comments: 1
Operations performed: 12
Github API rate used: 12

```
